File: Types/ReplayTypes/C/DiceType.cs
using System.ComponentModel;
using static BB2Replayer.Utils.Extensions;

namespace BB2Replayer.Replay.Types
{
	public enum BlockDiceType
	{
		Skull = 0, // Attacker Down
		BD = 1, // Both Down
		Push = 2, // Pushed
		Stumble = 3, // Defender Stumbles
		Pow = 4, // Defender Down
	}

	public enum DiceType
	{
		D3 = 1,
		D6,
		D8,
		Block,
		D68
	}
}

File: Types/ReplayTypes/Python/DiceType.py
from enum import Enum

class BlockDiceType(Enum):
    Skull = 0 # Attacker Down
    BD = 1 # Both Down
    Push = 2 # Pushed
    Stumble = 3 # Defender Stumbles
    Pow = 4 # Defender Down
    
class DiceType(Enum):
    D3 = 1
    # D6
    # D8
    # Block
    # D68